tests/StatsdServer.hpp

```diff
@@ -1,19 +1,28 @@
 #ifndef STATSD_SERVER_HPP
 #define STATSD_SERVER_HPP
 
-#include <netinet/in.h>
-#include <sys/socket.h>
-#include <unistd.h>
+// It might make sense to include this test class in the UDPSender header
+// it includes most of the cross platform defines etc that we need for socket io
+#include "cpp-statsd-client/UDPSender.hpp"
```

Comment:
because this class shares so much in common with UDPSender i didnt want to duplicate that code so i just included the header here. honestly i think it would kind of make sense to throw this class into the detail namespace in the UDPSender header but no worries if others feel strongly that we shouldnt.

Reply:
> it would kind of make sense to throw this class into the detail namespace in the UDPSender header

Not sure I understand what should be included where.

But adding the `UDPSender` header here is ok to avoid duplicating code.

Reply:
ah sorry for the confusion comment, i was again suggesting that we could just take the tiny bit of code that exists in this header and put it inside the `detail` namespace inside of the `UDPSender.hpp` header and completely delete this header. having a `UDP.hpp` with both a sender and receiver makes organizational sense to me. but again if you prefer to keep this test code out of the distributed header no problem. it was just a suggestion